client/src/helpers/APICalls/deleteImage.ts

```diff
@@ -0,0 +1,21 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+
+interface DeleteFetchOptions {
+  method: string;
+  credentials: RequestCredentials;
+}
+
+const deleteImage = async (_id: string): Promise<AuthApiData> => {
+  const fetchOptions: DeleteFetchOptions = {
+    method: 'DELETE',
+    credentials: 'include',
+  };
+
+  return await fetch(`/image/delete?` + new URLSearchParams({ _id }), fetchOptions)
```

Comment:
```suggestion
  return await fetch(`/image/delete` + new URLSearchParams({ _id }), fetchOptions)
```

Reply:
Making this change causes the following error
<img width="1920" alt="Screen Shot 2021-12-07 at 00 47 46" src="https://user-images.githubusercontent.com/18492009/144980428-7403e7b3-16cb-42f0-be0a-40325d0904db.png">
:

Reply:
Resolved this by adding data to request body.